dotCMS/src/main/resources/mssql.sql

```diff
@@ -2684,3 +2684,9 @@ create table storage_x_data (
 
 CREATE TABLE shedlock(name VARCHAR(64) NOT NULL, lock_until datetime NOT NULL,
                       locked_at datetime NOT NULL, locked_by VARCHAR(255) NOT NULL, PRIMARY KEY (name));
+
+create table variant (
+     id NVARCHAR(255) primary key,
+     name NVARCHAR(255) not null,
+     deleted tinyint not null,
```

Comment:
this should be called archived and init it

Reply:
done https://github.com/dotCMS/core/pull/22827/commits/9d53cd59ff07b5a9134e4d5df7d7741a7f0e4a60